pub/static/.htaccess

```
@@ -111,14 +122,17 @@ AddType application/xml xml
     ExpiresByType application/json "access plus 1 year"
 
     # Favicon, images, flash
-    <FilesMatch \.(ico|gif|png|jpg|jpeg|swf|svg)$>
+    <FilesMatch \.(ico|gif|png|jpg|jpeg|svg|svgz|webp|avif|avifs?)$>
```

I believe the `?` at the end is useless in this case as you listed both `avif` and `avifs` in the list:
```suggestion
    <FilesMatch \.(ico|gif|png|jpg|jpeg|svg|svgz|webp|avif|avifs)$>
```